internal/controllers/machinedeployment/machinedeployment_rolling.go

```diff
@@ -293,3 +299,25 @@ func (r *Reconciler) scaleDownOldMachineSetsForRollingUpdate(ctx context.Context
 
 	return totalScaledDown, nil
 }
+
+// cleanupDisableMachineCreateAnnotation will remove the disable machine create annotation from new MachineSets that were created during reconcileOldMachineSetsOnDelete.
+func (r *Reconciler) cleanupDisableMachineCreateAnnotation(ctx context.Context, newMS *clusterv1.MachineSet) (*clusterv1.MachineSet, error) {
+	log := ctrl.LoggerFrom(ctx)
```

Comment:
```suggestion
	log := ctrl.LoggerFrom(ctx, "MachineSet", klog.KObj(newMS))
```

Should we add the log pair here?